### PR TITLE
Adjust .swiftlint.yml according to new best practices

### PR DESCRIPTION
**Pull request by fredpi**
Should be merged together with https://github.com/JamitLabs/ProjLintTemplates/pull/4...

### CODE REVIEW

File: .swiftlint.yml [hunk 1]
```diff
@@ -101,9 +99,13 @@ file_name:
 #     - extension
 
 identifier_name:
+  max_length: 60
   excluded:
     - id
 
+nesting:
```
Comment (Jeehut) [inline .swiftlint.yml:108]: Please move this under `line_length` to keep the rule configurations sorted alphabetically.

**Review by Jeehut — CHANGES_REQUESTED** [review]
Nearly perfect. :)

File: .swiftlint.yml (hunk 1 above)
Comment (fredpi) [reply .swiftlint.yml:108]: @Dschee fixed it. ;)

**Review by Jeehut — APPROVED** [review]
Looks good now, thanks! 🎉

**Pull request merged**